config.js

```diff
@@ -15,4 +17,6 @@ config.usersExemptFromMaximum = process.env.EXEMPT_USERS?.split(",") || [
   "U8T585Y8J",
 ];
 
+config.initialGoldenRecognitionHolder = "UE1QRFSSY";
```

Comment:
I think this is fine, but just want to confirm that you are the expected initial user? Or is this for testing?

Reply:
Was intentional yea, was planning on giving it away right away though 🙂